anesthetic/weighted_pandas.py

```diff
@@ -45,6 +45,7 @@ class WeightedSeries(_WeightedObject, pandas.Series):
 
     def __init__(self, *args, **kwargs):
         w = kwargs.pop('w', None)
+        w = kwargs.pop('weight', w)
```

Comment:
Do we want to add a `DeprecationWarning` here, such that we can reduce this to one line in the future?